### DISCUSSION TITLE
I speak without a mouth and hear without ears. I have no body, but I come alive with wind. What am I?

### DISCUSSION THREAD

**Discussion by meviru** [meviru/badge-hunting#8] [Galaxy Trivia] [answered]
Puzzle: I speak without a mouth and hear without ears. I have no body, but I come alive with wind. What am I?

**Comment by Choksirahul** [comment, marked as answer]
An echo!

**Reply by meviru** [reply]
Congratulations!! 👏👏 You've just earned the [Galaxy Brain](https://github.com/Choksirahul?tab=achievements&achievement=galaxy-brain) badge. Keep it up, and remember to give the repository a star to show your support! 🎉
